scripts/review-checklist.sh

```
@@ -0,0 +1,243 @@
+  for re in "${rules[@]}"; do
+    [[ -z "$re" ]] && continue
+    if [[ "$file" =~ $re ]]; then
+      return 0
+    fi
+  done
+  return 1
+}
+
+matches_any_regex_rule() {
+  local file="$1"
+  local rules="$2"
```

Shellcheck (0.11.0)

[warning] Variable was used as an array but is now assigned a string.

(SC2178)